### PR TITLE
JBIDE-21461: Upgrades to Forge 3.0.0.Beta2

### PR DESCRIPTION
**Pull request by gastaldi**
@maxandersen @fbricon @alexeykazakov can anyone review this?

### CODE REVIEW

File: plugins/org.jboss.tools.forge.runtime/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -37,6 +37,7 @@ Export-Package: bootpath,
    org.jboss.forge.addon.projects,
    org.jboss.forge.addon.dependencies,
    org.jboss.forge.addon.resource",
+ org.jboss.forge.addon.projects.stacks,
```
Comment (fbricon) [inline plugins/org.jboss.tools.forge.runtime/META-INF/MANIFEST.MF]: why is this necessary? I don't see other references to stacks in that PR

Comment (gastaldi) [inline plugins/org.jboss.tools.forge.runtime/META-INF/MANIFEST.MF]: If I don't declare it, I get a ClassNotFoundException. Probably because org.jboss.forge.addon.projects.Project has a Stack getStack() method.

**Comment by fbricon** [comment]
ok to apply

**Comment by gastaldi** [comment]
Thanks :+1:
